engine/src/main/java/org/hibernate/validator/internal/metadata/provider/AnnotationMetaDataProvider.java

```diff
@@ -278,8 +278,8 @@ private UnwrapMode unwrapMode(ExecutableElement executable, boolean typeArgument
 		return constraints;
 	}
 
-	private UnwrapMode unwrapMode(boolean typeArgumentAnnotated, boolean notIterable, UnwrapValidatedValue unwrapValidatedValue) {
-		if ( unwrapValidatedValue == null && typeArgumentAnnotated && notIterable ) {
+	private UnwrapMode unwrapMode(boolean typeArgumentAnnotated, boolean notIndexable, UnwrapValidatedValue unwrapValidatedValue) {
+		if ( unwrapValidatedValue == null && typeArgumentAnnotated && notIndexable ) {
```

Comment:
Can we make it a "positive" variable name? It's easier to grok that way, and more idiomatic: `!indexable`.